phpmyadmin/config.yaml

```diff
@@ -19,3 +19,4 @@ services:
 hassio_api: true
 schema:
   log_level: list(trace|debug|info|notice|warning|error|fatal)?
+  upload_limit: int?
```

Comment:
Should we make it just... big? Instead of making it configurable?

Reply:
I did think about it, but my main concern was "I uploaded a 10Gb file and my Pi disk is full)", I really wanted to make it a decision to change it.  Plus I also think the use cases are minimal.

Reply:
True 👍